connectivity/FEATURE_BLE/libraries/TARGET_CORDIO_LL/mbed_lib.json

```diff
@@ -56,5 +56,11 @@
             "value": 1,
             "macro_name": "CHCI_TR_CUSTOM"
         }
+    },
+    "target_overrides": {
```

Comment:
Ideally this would be in the NRF5X driver but it is not possible du to the build system limitations.

Reply:
Can we fix it in CMake ? If yes, we should